src/utils/validations/experience.js

```diff
@@ -0,0 +1,8 @@
+export const experienceField = (value) => {
```

Comment:
Make it general so that you can reuse it for any field. Just pass min and max values.